cookbook/control_plane/run_task.py

```diff
@@ -1,3 +1,8 @@
 """
 Run a Single Task
```

Comment:
why is this under control plane?

Reply:
I agree, should it not be `executions` and then 
run_task - local and remote?